README.md

```diff
@@ -100,3 +100,44 @@ yarn test
 Frames are expected to follow the Open Frames Standard https://github.com/open-frames/standard
 
 
+
+# Release Processes
+
+### Main Branch
+Represents the current production code.
+
+### Release Branches
+Each release branch is based off of `main` or the release branch before it. It is used to prepare and stabilize the code for a specific release version (e.g., `release/2.0.8`).
```

Comment:
When would each scenario occur?

ie based off of release vs main?

Reply:
Almost always be based off of a release branch
The only time you would use main is in the exceptions section